src/ucan-cloud/store/state/index.ts

```diff
@@ -2,10 +2,10 @@ import { runtimeFn } from "@adviser/cement";
 
 export default async function store(name: string) {
   if (runtimeFn().isNodeIsh) {
-    const n = await import("./node.js");
+    const n = await import("../../../ucan-cloud/store/state/node.js");
```

Comment:
why is this needed the file is in state.

Reply:
It’s a hack, but it works. It’s so that the `resolve` plugin in tsup doesn’t replace similar imports in our dependencies (ie. by using this path we’re making it more unique)

There definitely has to be a better solution. But it’s the only quick fix I found to work 🤷‍♂️

Reply:
@mschoch I want to see the build error that causes this change.
there is something very unusual which could cause other unwanted side effects.

Reply:
@mabels I want to try and help get you that without redoing everything we did on Friday.  But you haven't been able to be more specific, so it will have wait until Monday,.